cmd/apiserver/app/server/util.go

```diff
@@ -148,7 +149,7 @@ func buildGenericConfig(s *ServiceCatalogServerOptions) (*genericapiserver.Recom
 
 		// TODO: we need upstream to package AlwaysAdmit, or stop defaulting to it!
```

Comment:
@derekwaynecarr @pmorie unrelated note: is this TODO still valid? I can't find "AlwaysAdmit" anywhere in the code.
Asking because in this PR there is an authorizer introduced that might affect this TODO?

Reply:
AlwaysAdmit is in k8s.io/apiserver, but not used by default (anymore?)